lib/cc/engine/coffeelint_results.rb

```diff
@@ -3,23 +3,37 @@
 module CC
   module Engine
     class CoffeelintResults
+      DEFAULT_CONFIG_PATH = "/code/coffeelint.json".freeze
+
       def initialize(directory, files, config)
         @directory = directory
-        @config = config
         @files = files
+        @config = config
       end
 
       def results
         return [] if @files.empty?
 
         escaped_files = Shellwords.join(@files)
         cmd = File.expand_path("../../../node_modules/.bin/coffeelint", __dir__)
-        cmd << " -f #{@config}" if @config
+        cmd << " -f #{config_path}" if config_path
         cmd << " -q --reporter raw #{escaped_files}"
         Dir.chdir(@directory) do
           JSON.parse(`#{cmd}`)
         end
       end
+
+      private
+
+      attr_reader :config
```

Comment:
Sorry if I wasn't clear before - I think this is the opportunity for all 3 ivars to become `attr_reader`s and accessed that way. It seems worse to only have one